t/api/04-jobs.t

```diff
@@ -371,4 +371,13 @@ $get = $t->get_ok('/api/v1/jobs', form => $job_properties);
 is($get->tx->res->json->{jobs}->[1]->{group},    'opensuse test');
 is($get->tx->res->json->{jobs}->[1]->{priority}, 42);
 
+$get = $t->get_ok('/api/v1/jobs/99926')->status_is(200);
+is($get->tx->res->json->{job}->{group}, 'opensuse', 'current group is opensuse');
```

Comment:
There's a test method you can use for this too. http://mojolicious.org/perldoc/Test/Mojo#json_is
```perl
$t->json_is('/job/group' => 'opensuse', 'current group is opensuse');
```